src/components/InputForm.jsx

```diff
@@ -10,65 +9,43 @@ export default function InputForm({ handleChange, formData, hexColor }) {
             <div>
               <button
                 style={{ width: "130px" }}
-                className="rounded-full border-white border-2 scheme-button"
-              >
+                className="rounded-full border-white border-2 scheme-button">
                 Analogue
               </button>
             </div>
             <div>
               <button
                 style={{ width: "130px" }}
-                className="rounded-full border-white border-2 scheme-button"
-              >
+                className="rounded-full border-white border-2 scheme-button">
                 Triadic
               </button>
             </div>
             <div>
               <button
                 style={{ width: "150px" }}
-                className="rounded-full border-white border-2 scheme-button"
-              >
+                className="rounded-full border-white border-2 scheme-button">
                 Complimentary
               </button>
             </div>
           </div>
           <div className="shadow-pink-50 flex justify-center mx-2.5 scheme-hexCodes-container">
             <div
               style={{ background: "#31D7BE", borderRadius: "40px 0 0 40px" }}
-              className="flex justify-center items-center border-white border-2 hex-codes"
-            >
+              className="flex justify-center items-center border-white border-2 hex-codes">
               {hexColor}
             </div>
             <div
               style={{ background: "#BE31D7" }}
-              className="flex justify-center items-center border-white border-2  hex-codes"
-            >
+              className="flex justify-center items-center border-white border-2  hex-codes">
               {hexColor}
             </div>
             <div
               style={{ background: "#D7BE31", borderRadius: " 0 40px 40px 0" }}
-              className="flex justify-center items-center border-white border-2 hex-codes"
-            >
+              className="flex justify-center items-center border-white border-2 hex-codes">
               {hexColor}
             </div>
           </div>
         </div>
-
-        {/* <label htmlFor="usage">
-          Describe where do you plan to use this color
-        </label> */}
-        <div>
-          <textarea
-            className="rounded-xl shadow-lg p-3"
-            cols="40"
-            rows="1"
-            placeholder="usage"
-            name="usage"
-            onChange={handleChange}
-            id="usage"
-            value={formData.usage}
-          />
-        </div>
       </form>
```

Comment:
I can't see an opening tag for this form.

Currently, the required change is not working due to the textarea not being within the form element.

Reply:
Nvm I see it on line 7.

I am still wondering why I can submit the form even if it is empty...Going to dig more into this.

Reply:
This form is actually fine to be left alone - It is a semantic tag even though we aren't technically using it as a form. We could choose to remove the `<form>` element tag or remove it.

The problem with required not working is due to the same reason, but in a another file.